2014/09/29 00:54:53 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.tbl_person' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_person`.
in Z:\home\yii\www\index.php (13)
2014/09/29 00:57:37 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.tbl_person' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_person`.
in Z:\home\yii\www\index.php (13)
2014/09/30 19:56:02 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.sity' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `sity`.
in Z:\home\yii\www\index.php (13)
2014/09/30 19:57:14 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.sity' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `sity`.
in Z:\home\yii\www\index.php (13)
2014/09/30 19:57:26 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.sity' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `sity`.
in Z:\home\yii\www\index.php (13)
2014/09/30 19:57:38 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.sity' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `sity`.
in Z:\home\yii\www\index.php (13)
2014/09/30 19:59:05 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.sity' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `sity`.
in Z:\home\yii\www\index.php (13)
2014/09/30 19:59:18 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.sity' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `sity`.
in Z:\home\yii\www\index.php (13)
2014/09/30 19:59:29 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.sity' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `sity`.
in Z:\home\yii\www\index.php (13)
2014/09/30 21:31:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "person".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('person')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('person')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/person/ind...')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=city/person/index
---
2014/09/30 21:32:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "city person/index".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city person/ind...')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=city%20person/index
---
2014/10/01 20:23:41 [error] [exception.CException] exception 'CException' with message 'Property "CBelongsToRelation.0" is not defined.' in Z:\home\yii\YiiRoot\framework\base\CComponent.php:173
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\db\ar\CActiveRecord.php(1998): CComponent->__set('0', 'joinType')
#1 Z:\home\yii\YiiRoot\framework\db\ar\CActiveRecord.php(2408): CBaseActiveRelation->__construct('City', 'City ', 'id', Array)
#2 Z:\home\yii\YiiRoot\framework\db\ar\CActiveRecord.php(2389): CActiveRecordMetaData->addRelation('City', Array)
#3 Z:\home\yii\YiiRoot\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Person))
#4 Z:\home\yii\YiiRoot\framework\db\ar\CActiveRecord.php(658): CActiveRecord->getMetaData()
#5 Z:\home\yii\YiiRoot\framework\db\ar\CActiveRecord.php(1601): CActiveRecord->getTableSchema()
#6 Z:\home\yii\YiiRoot\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#7 Z:\home\yii\YiiRoot\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#8 Z:\home\yii\YiiRoot\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#9 Z:\home\yii\YiiRoot\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#10 Z:\home\yii\YiiRoot\framework\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#11 Z:\home\yii\YiiRoot\framework\zii\widgets\CListView.php(211): CBaseListView->init()
#12 Z:\home\yii\YiiRoot\framework\web\CBaseController.php(147): CListView->init()
#13 Z:\home\yii\YiiRoot\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.CLi...', Array)
#14 Z:\home\yii\www\protected\views\person\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#15 Z:\home\yii\YiiRoot\framework\web\CBaseController.php(126): require('Z:\home\yii\www...')
#16 Z:\home\yii\YiiRoot\framework\web\CBaseController.php(95): CBaseController->renderInternal('Z:\home\yii\www...', Array, true)
#17 Z:\home\yii\YiiRoot\framework\web\CController.php(869): CBaseController->renderFile('Z:\home\yii\www...', Array, true)
#18 Z:\home\yii\YiiRoot\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#19 Z:\home\yii\www\protected\controllers\PersonController.php(128): CController->render('index', Array)
#20 Z:\home\yii\YiiRoot\framework\web\actions\CInlineAction.php(49): PersonController->actionIndex()
#21 Z:\home\yii\YiiRoot\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#22 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#23 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(40): CFilterChain->run()
#24 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#25 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#26 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#27 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#28 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#29 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('')
#30 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('person')
#31 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#32 Z:\home\yii\www\index.php(13): CApplication->run()
#33 {main}
REQUEST_URI=/index.php?r=person
---
2014/10/02 18:13:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/admin".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user/admin
---
2014/10/02 18:14:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2014/10/02 19:09:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "city".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('city')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('city')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('person/city')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=person/city
---
2014/10/02 19:45:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "city".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('city')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('city')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('person/city')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=person/city
---
2014/10/02 19:46:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "relationTbl".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('relationTbl')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('relationTbl')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/relationTb...')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=city/relationTbl
---
2014/10/02 19:46:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "relationTbl".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('relationTbl')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=relationTbl
---
2014/10/02 19:53:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('login')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('login')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('person/login')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=person/login
---
2014/10/02 19:54:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "Login".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('Login')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('Login')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('person/Login')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=person/Login
---
2014/10/02 20:24:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index=2".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('index=2')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('index=2')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/index=2')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=city/index=2
---
2014/10/02 20:25:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "2".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('2')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=city&r=2
---
2014/10/03 19:48:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "findByPk".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('findByPk')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('findByPk')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/findByPk')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=city/findByPk
---
2014/10/03 20:15:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "citн".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('cit??')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=cit%D0%BD
---
2014/10/03 20:35:36 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=city/tbl
---
2014/10/03 20:43:32 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=city/tbl
---
2014/10/03 20:49:15 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=city/tbl
---
2014/10/03 20:49:37 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=city/tbl
---
2014/10/03 20:50:27 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=city/tbl
---
2014/10/03 20:57:07 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=city/tbl
---
2014/10/03 21:26:21 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=city/tbl
---
2014/10/03 21:57:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "=city/tbl".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('=city/tbl')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/=city/tbl
---
2014/10/03 21:57:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "=city/tbl".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('=city/tbl')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/=city/tbl
---
2014/10/03 22:00:16 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/city/tbl
HTTP_REFERER=http://yii/index.php/site/login
---
2014/10/03 22:03:07 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/city/tbl
HTTP_REFERER=http://yii/index.php/site/login
---
2014/10/04 16:33:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index/tbl".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('index/tbl')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index/tbl
---
2014/10/04 16:34:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/10/04 16:34:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "tbl".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('tbl')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/tbl
---
2014/10/04 16:35:15 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/city/tbl
HTTP_REFERER=http://yii/index.php/site/login
---
2014/10/04 16:35:44 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('Tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/Tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/city/Tbl
---
2014/10/04 16:35:48 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('Tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/Tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/city/Tbl
---
2014/10/04 17:14:23 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('Tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/Tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/city/Tbl
---
2014/10/04 17:14:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "tbl".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('tbl')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/tbl
---
2014/10/04 17:15:07 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/city/tbl
---
2014/10/04 17:21:14 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 Z:\home\yii\YiiRoot\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 Z:\home\yii\YiiRoot\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 Z:\home\yii\YiiRoot\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 Z:\home\yii\YiiRoot\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 Z:\home\yii\YiiRoot\framework\web\CController.php(291): CFilterChain->run()
#6 Z:\home\yii\YiiRoot\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('tbl')
#8 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('city/tbl')
#9 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 Z:\home\yii\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/city/tbl
---
2014/10/04 21:04:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/10/04 21:04:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/10/05 21:45:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/10/05 21:55:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "personcity".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('personcity')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/personcity
---
2014/10/05 22:07:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "views/PersonCity/index".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('views/PersonCit...')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/views/PersonCity/index
---
2014/10/05 22:07:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "views/PersonCity/index".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('views/PersonCit...')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/views/PersonCity/index
---
2014/10/05 22:08:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "views/PersonCity/index.php".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('views/PersonCit...')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/views/PersonCity/index.php
---
2014/10/05 22:08:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('index.php')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('PersonCity/inde...')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/PersonCity/index.php
---
2014/10/05 22:09:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('index.php')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('personCity/inde...')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/personCity/index.php
---
2014/10/05 22:09:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "personсity/index.php".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('person??ity/ind...')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/person%D1%81ity/index.php
---
2014/10/05 22:09:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "personсity/index.php".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('person??ity/ind...')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/person%D1%81ity/index.php
---
2014/10/05 22:09:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "personсity/index.php".' in Z:\home\yii\YiiRoot\framework\web\CWebApplication.php:286
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('person??ity/ind...')
#1 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Z:\home\yii\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/person%D1%81ity/index.php
---
2014/10/05 22:11:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in Z:\home\yii\YiiRoot\framework\web\CController.php:483
Stack trace:
#0 Z:\home\yii\YiiRoot\framework\web\CController.php(270): CController->missingAction('index.php')
#1 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 Z:\home\yii\YiiRoot\framework\web\CWebApplication.php(141): CWebApplication->runController('personCity/inde...')
#3 Z:\home\yii\YiiRoot\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 Z:\home\yii\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/personCity/index.php
---
